circle.yml

```diff
@@ -4,8 +4,7 @@ machine:
   node:
     version: v5.5.0
   environment:
-    NODE_012_VERSION: v0.12.0
-    NODE_4_VERSION: v4.2.2
+    NODE_4_VERSION: v4.5.0
```

Comment:
@minasmart @richgilbank how do you feel about this? I know in pervious discussions there was no issue moving away from v0.12.0. Doing this basically makes it so that we can use some es2015 features in Node for the build tool. If 0.12.0 is mandatory maybe we can transpile the `scripts/` dir and then run those but seems a bit yucky to do that.